### DISCUSSION TITLE
Question guys

### DISCUSSION THREAD

**Discussion by Tommylovespython** [community/community#45198] [New to GitHub] [answered]
Hi guys I’m new here and wonder how to earn two badges. Quick draw and Yolo. I’m barely have my first repository. I’m a beginning coder and will hopefully grow. Tia.

**Comment by jge162** [comment, marked as answer]
You earn the yolo badge when you ask for a review but do not wait for review approval and commit pull request.  Hope that helps…

Quick draw is easy to earn, you open an issue and close it within 5 mins of opening it up. 

Hope that makes sense. If not LMK thanks…

**Reply by Tommylovespython** [reply]
Ok got the quick draw to work. Ty

**Reply by Tommylovespython** [reply]
How do I do the Yolo?

**Reply by jge162** [reply]
as mentioned above...

**Reply by Tommylovespython** [reply]
Ah ok. I’m not ready to do that. But thanks. Got my quick draw.Ty Ty
